spec/spec_helper.rb

```diff
@@ -0,0 +1,32 @@
+$LOAD_PATH.unshift File.expand_path('../lib', __FILE__)
+
+MINIMUM_COVERAGE = 80
+
+unless ENV['COVERAGE'] == 'off'
```

Comment:
Can we move coverage to support/coverage_loader.rb as in all other gems now (I just did this yesterday for 10-20 gems)

Also see how they do SimpleCov.formatter for the nicer syntax.